src/common/types.ts

```diff
@@ -71,11 +71,9 @@ export interface PaginatedResults<T> {
   results: T[];
 }
 
-type Amount = number | '';
-
 export interface TransactionFormValues {
-  amount: Amount;
+  amount: number | string;
```

Comment:
I don't think this type needs to be here, could be nearer where it's used in the feature?

Reply:
It is used in both the `send-tokens-form` and in `sign-transaction` and shared in the global `local` form state in the store. Is that not the right way to do it? I can separate and not share.